Gameplay/ui/DebugManager.cpp

```diff
@@ -247,8 +247,8 @@ void Hachiko::Scripting::DebugManager::HandleButtonInteraction()
 	if (_spawn_enemy->IsSelected())
 	{
 		HE_LOG("_spawn_enemy pressed");
-		UID* enemy_uid = new UID(11363594999076676195);
-		GameObject* enemy = GameObject::Instantiate(enemy_uid, game_object->parent->FindDescendantWithName("Enemies"));
+		UID enemy_uid = UID(11363594999076676195);
```

Comment:
```suggestion
		UID enemy_uid = 11363594999076676195);
```
This is even better probably.

Reply:
Thank you so much for changing how it was before btw, wtf is `new` for a `long long unsigned` hehe